pipes/WDL/tasks/tasks_nextstrain.wdl

```diff
@@ -376,6 +376,68 @@ task nextstrain_ncov_defaults {
     }
 }
 
+task nextstrain_ncov_sanitize_gisaid_data {
+    meta {
+        description: "Sanitize data downloaded from GISAID for use in Nextstrain/augur. See: https://nextstrain.github.io/ncov/data-prep#curate-data-from-the-full-gisaid-database"
+    }
+
+    input {
+        File sequences_gisaid_fasta
+        File metadata_gisaid_tsv
+
+        String? prefix_to_strip = "hCoV-19/"
+
+        String nextstrain_ncov_repo_commit = "183e94fd5ee73be97d66b7b7d90b167146fa0752"
+        String docker                      = "nextstrain/base:build-20210413T201712Z"
+    }
+
+    parameter_meta {
+        sequences_gisaid_fasta: {
+          description: "Multiple sequences downloaded from GISAID",
+          patterns: ["*.fasta","*.fasta.xz","*.fasta.gz"]
+        }
+        metadata_gisaid_tsv: {
+            description: "Tab-separated metadata file for sequences downloaded from GISAID and passed in via sequences_gisaid_fasta.",
+            patterns: ["*.txt", "*.tsv","*.tsv.xz","*.tsv.gz"]
+        }
+        prefix_to_strip: {
+            description: "String prefix to strip from sequence IDs in both the input fasta and metadata files"
+        }
+    }
+
+    String out_basename = basename(basename(basename(basename(sequences_gisaid_fasta, '.xz'), '.gz'), '.tar'), '.fasta')
+    command {
+        set -e
+        ncov_path_prefix="/nextstrain/ncov"
+        wget -q "https://github.com/nextstrain/ncov/archive/~{nextstrain_ncov_repo_commit}.tar.gz"
+        mkdir -p "$ncov_path_prefix"
+        tar -xf "~{nextstrain_ncov_repo_commit}.tar.gz" --strip-components=1 -C "$ncov_path_prefix"
+
+        python3 "$ncov_path_prefix/scripts/sanitize_sequences.py" \
+        --sequences "~{sequences_gisaid_fasta}" \
+        ~{"--strip-prefixes=" + prefix_to_strip} \
+        --output "~{out_basename}_sequences_sanitized_for_nextstrain.fasta.gz"
+
+        python3 "$ncov_path_prefix/scripts/sanitize_metadata.py" \
+        --metadata "~{metadata_gisaid_tsv}" \
+        --parse-location-field Location \
+        --rename-fields 'Virus name=strain' 'Accession ID=gisaid_epi_isl' 'Collection date=date' 'Clade=GISAID_clade' 'Pango lineage=pango_lineage' 'Host=host' 'Type=virus' 'Patient age=age' \
+        ~{"--strip-prefixes=" + prefix_to_strip} \
+        --output "~{out_basename}_metadata_sanitized_for_nextstrain.tsv.gz"
+    }
+    runtime {
+        docker: docker
+        memory: "7 GB"
+        cpu:   1
+        disks:  "local-disk 50 HDD"
```

Comment:
I'm a bit surprised we can get away with 50GB disk, but given that cloud disk speeds are proportional to disk size, it seems we should ask for more anyway. Or maybe just get around the whole thing with a `disks: "local-disk 375 LOCAL"` to max out the I/O on a GCE VM.